iree-jax/benchmark/benchmark_model.py

```diff
@@ -155,32 +191,33 @@ def run_framework_benchmark(model_name: str, model_class: Any, batch_size: int,
       f"\n\n--- {model_name} {args.benchmark_id} -------------------------------------"
   )
 
-  batch_size = model_definition.input_batch_size
   benchmark_definition = {
       "benchmark_id": args.benchmark_id,
       "benchmark_name": model_definition.name,
       "framework": str(model_definition.meta_model.framework_type),
       "data_type": str(model_definition.meta_model.data_type),
-      "batch_size": batch_size,
+      "batch_size": model_definition.input_batch_size,
       "inputs": model_definition.inputs.tensor_dimensions,
       "outputs": model_definition.outputs.tensor_dimensions,
       "compiler": "xla",
       "device": args.device,
       "tags": model_definition.meta_model.tags + model_definition.tags,
   }
 
+  inputs = retrieve_data(model_definition.inputs, args.cache_dir)
+  expected_outputs = retrieve_data(model_definition.outputs, args.cache_dir)
```

Comment:
We would like to be able to download the artifacts in a separate step, so we don't worry about the network issue during benchmarking. No need to do in this change.

Reply:
Yeah we should create a helper that will download all artifacts given a configuration.